pkg/fakerp/config.go

```diff
@@ -55,44 +72,36 @@ func GetPluginConfig() (*api.PluginConfig, error) {
 	if err != nil {
 		return nil, err
 	}
+	template.Certificates.GenevaLogging.Cert = logCert
```

Comment:
I'd be tempted to ditch the separate files and put it all in the template.yaml

Reply:
Think about ease of onboarding new people

Reply:
This means we will need to do the conversion to `base64` and all file becomes sensitive information and we can commit the only example or redacted one.

Reply:
oh, yuck, I see, the issue is being able to check in the basics for MSFT, but keep the secrets separate.  Can you think how to try and make this as neat as poss for us and for MSFT?

Reply:
We could have a small helper to populate file locally from example file. 
So we run it to "bootstrap" config once. This would basically do same as current code is doing.

Reply:
moved back to RP